### ISSUE TITLE
imProving Agent doesn't support QNode IDs without QNode categories

### ISSUE THREAD

**Issue by tursynay** [NCATSTranslator/minihackathons#11]
Context for the issue:
When the below query was ran, query result showed errors for Improve Agent. There seemed to be a registration issue with ARS, and this result needs to be checked again once registration is complete. 

Query:
qg = {
"nodes": {
"n0": {
"categories": [
"biolink:ChemicalSubstance"
],
"name": "Chemical Substance"
},
"n1": {
"name": "EGFR",
"ids": ["NCBIGene:1956"]
}
},
"edges": {
"e0": {
"subject": "n0",
"object": "n1",
"predicates": [
"biolink:decreases_abundance_of",
"biolink:decreases_activity_of",
"biolink:decreases_expression_of",
"biolink:decreases_synthesis_of",
"biolink:increases_degradation_of",
"biolink:disrupts",
"biolink:negatively_regulates_entity_to_entity"
]
}
}
}

Error message:
https://arax.ncats.io/?source=ARS&id=90080f07-808c-47db-a042-21a9ce6f4281

**Comment by tursynay** [comment]
@baranzini-lab, wanted to reach out about this as we have the mini-hackathon this week. Was there any update on the issue?

**Comment by tursynay** [comment]
We have another mini-hack tomorrow, any updates @suihuang-ISB , @baranzini-lab?

**Comment by vgardner-renci** [comment]
mProving Agent requires that qNodes with identifier also specify a biolink category; returning empty message...
Query unprocessable

**Comment by vgardner-renci** [comment]
@brettasmi is taking a look

**Comment by brettasmi** [comment]
@vgardner-renci, can we change the title of this to the more accurate "imProving Agent doesn't support QNode IDs without QNode categories" OR create a new issue with that title?

Thanks!

**Comment by vgardner-renci** [comment]
@brettasmi name edited for clarity

**Comment by brettasmi** [comment]
I've created an issue for this that you can see in the notice above 👆

**Comment by tursynay** [comment]
is this out of scope for improve agent?

**Comment by jenndoug** [comment]
Known issue; inter-related with A8 and A9.  Big work for us.  Timeline before September Relay (9/27-10/1)

**Comment by brettasmi** [comment]
Fixed.
